### PR TITLE
Move plural to dedicated nodes

### PR DESCRIPTION
**Pull request by DavidSpek**
Work in progress

## Summary
Move the processes running Plural to the new dedicated node group.

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: plural/terraform/aws/terraform.tfvars [hunk 1]
```diff
@@ -5,4 +5,9 @@ plural_assets_bucket = {{ .Values.assets_bucket | quote }}
 plural_images_bucket = {{ .Values.images_bucket | quote }}
 plural_namespace = {{ .Namespace | quote }}
 cluster_name = {{ .Cluster | quote }}
-shell_subnet_ids = {{ $bootstrap.cluster_worker_private_subnet_ids | toJson }}
\ No newline at end of file
+shell_subnet_ids = {{ $bootstrap.cluster_worker_private_subnet_ids | toJson }}
+node_role_arn = {{ (index $bootstrap.node_groups 0).node_role_arn | quote }}
```
Comment (michaeljguarino) [inline plural/terraform/aws/terraform.tfvars:9]: how does the shell nodes get this?  It's probably reusable

Comment (DavidSpek) [reply plural/terraform/aws/terraform.tfvars:9]: Inzong think it does. It’s not passing a value for the arn in any case.

Comment (DavidSpek) [reply plural/terraform/aws/terraform.tfvars:9]: Which might be good from a security standpoint.

Comment (michaeljguarino) [reply plural/terraform/aws/terraform.tfvars:9]: looks like it's done within a module library submodule here: https://github.com/pluralsh/module-library/blob/main/terraform/eks-node-groups/multi-az-node-group/main.tf#L9

**Pull request merged**